addons/mail/static/tests/legacy/helpers/mock_server/models/mail_notification.js

```diff
@@ -51,7 +51,7 @@ patch(MockServer.prototype, {
                 notification_status: notification.notification_status,
                 failure_type: notification.failure_type,
                 persona: partner
-                    ? { id: partner.id, displayName: partner.display_name, type: "partner" }
+                    ? { id: partner.id, name: partner.name, type: "partner" }
```

Comment:
why do we still to modify the legacy tests?

Reply:
We need to maintain them or remove them I think.